include/vctr/vctr.h

```diff
@@ -163,9 +163,11 @@
 #include "Expressions/Core/Mean.h"
 #include "Expressions/Core/Sum.h"
 
+#include "Expressions/Complex/Angle.h"
 #include "Expressions/Complex/Conjugate.h"
 #include "Expressions/Complex/Imag.h"
 #include "Expressions/Complex/Real.h"
+#include "Expressions/Complex/PowerSpectrum.h"
```

Comment:
If you want to maintain alphabetic order, `PowerSpectrum.h` should come before `Real.h`.

Reply:
Yeah, good point, this is currently a wild mixture between a logical and an alphabetical order. I want to do a general cleanup for stuff like that (also renaming some source files, fixing the case of the namespaces...) separately